frontend/packages/pipelines-plugin/integration-tests/support/step-definitions/common/common.ts

```diff
@@ -1,34 +1,33 @@
 import { Given, When, Then } from 'cypress-cucumber-preprocessor/steps';
 import { guidedTour } from '../../../../../integration-tests-cypress/views/guided-tour';
-import {
-  navigateTo,
-  perspective,
-  projectNameSpace,
-} from '@console/dev-console/integration-tests/support/pages/app';
 import { nav } from '../../../../../integration-tests-cypress/views/nav';
```

Comment:
same here. use `@console/cypress-integration-tests`

Reply:
Fix this in all other places in this PR, looks like all the reference are having relative imports.

Reply:
updated